modules/tools/record_play/rtk_player.py

```diff
@@ -168,6 +168,25 @@ def closest_time(self):
 
         return closest_time
 
+    def next_gear_switch_time(self, start, end):
+        for i in range(start, end):
+            # trajectory with gear switch
+            # include gear_neutral at the beginning of a trajectory
+            if((self.data['gear'][i] == 1 or self.data['gear'][i] == 2)
+                    and (self.data['gear'][i + 1] != self.data['gear'][i]) ):
+                print("enter i in while loop: [", i, "]")
```

Comment:
please use self.logger.info/debug to show information instead of print

Reply:
fixed. Thank you.